feat(uid) Adding modal documentation (#605)
bonitasoft/bonita-doc: md/uid-modal-tutorial.md

# How to create a modal window using CSS

::: warning

> **:fa-exclamation-triangle: Warning:** This tutorial is deprecated after 7.8 because of the introduction of the modal container. For more information, read the [modal container](widgets.md#modal-widget) page.

Reviewer: ```suggestion
**:fa-exclamation-triangle: Warning:** This tutorial is deprecated starting with Bonita 7.8 because of the introduction of the modal container as a provided container. For more information, read the [modal container](widgets.md#modal-widget) page.
```

> **:fa-exclamation-triangle: Warning:** This tutorial is deprecated after 7.8 because of the introduction of the modal container. Please read more about the modal on the widgets page : [Modal Container](widgets.md)

Reviewer: ```suggestion
**:fa-exclamation-triangle: Warning:** This tutorial is deprecated after 7.8 because of the introduction of the modal container. For more information, read the [modal container](widgets.md) page.
```

Reviewer: You need to place an anchor in the widget page and link this warning directly to the anchor.

:::

> A nice aspect of the UI designer is the way it takes advantage of the WEB technologies letting you put the style of your page where it belong.  

Reviewer: ```suggestion
A nice aspect of the UI Designer is the way it takes advantage of the WEB technologies, letting you put the style of your page where it belongs.  
```
